pkg/apis/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rook/rook/pkg/apis
 
-go 1.24.10
+go 1.24.0
+
+toolchain go1.24.6
```

Comment:
toolchain can be used to provide the maximum required version.

Reply:
done